src/WebJobs.Extensions.DurableTask/AzureStorageDurabilityProvider.cs

```diff
@@ -200,6 +203,17 @@ internal static OrchestrationInstanceStatusQueryCondition ConvertWebjobsDurableC
         }
 
 #if !FUNCTIONS_V1
+
+        internal DurableTaskMetricsProvider GetMetricsProvider(
+            string functionName,
+            string hubName,
+            CloudStorageAccount storageAccount,
+            ILogger logger,
+            DisconnectedPerformanceMonitor performanceMonitor = null)
```

Comment:
I recommend not using default parameter values for internal methods. I'd rather have the caller explicitly pass null when making internal method calls.

Reply:
Makes sense. I realized after this comment that this parameter was always null, so I just removed it anyways.